docs/docs/docker/tagging-docker-images.mdx

```diff
@@ -131,6 +131,7 @@ docker_image(
 The default placeholders are:
 
 - `{directory}`: The directory the docker_image's BUILD file is in.
```

Comment:
While in the vicinity, I think we could tweak the wording here in an attempt to make it more clear (I too was confused by this.. and I wrote it!)

Perhaps:
```suggestion
- `{directory}`: The folder name of the docker_image's BUILD file.
```